doc/pages/MAINPAGE.md

```diff
@@ -4,4 +4,95 @@ To get started on the project, all the instructions are linked into the README:
+
+### Two families of MCDA methodologies
+MCDA methods are generally classified in two families. The first family encompasses methods based on multi-attribute value theory (MAVT) (Keeney and Raiffa, 1976) while the second includes methods based on outranking relations (Roy, 1991). 
+
+They are many outranking methods but Roy and Bouyssou (1993) developed ELECTRE TRI which allows to deal with sorting problems.
+**As matter of fact, majority rule sorting models are a simplified version of ELECTRE TRI.**
+
+# MCDA Definition 
+
+**Criterion** and **alternative** are two important notions in MCDA. We recall here their definitions.
+In MCDA, a decision maker is often confronted to different solutions to a problem. These solutions are also called alternatives or actions.
+
+*An **alternative** “a” is the representation of a possible contribution to the comprehensive decision that can be considered autonomously with respect to the decision process development state and that can serve as an application point for the decision aid. The application point is, then, sufficient to characterize “a”.*
+
+In MCDA, the alternatives are evaluated on multiple attributes, called criteria. A criterion is a function that assigns an evaluation to each alternative involved in the decision problem. The coding of a criterion is often made such that the preference either increases or decreases as a function of the criterion value. The preference scale associated with a criterion should be monotone and each criterion should have a preference direction: the value of the criterion should be either minimized or maximized. For instance, in the context of choosing a car, disregarding all other criteria, a DM usually prefers a cheap car to an expensive one. 
+
+*A **criterion** is a function $$g_j$$ assigning a value to each alternative of a decision problem such that it becomes possible for a DM to perform pairwise comparisons and express a preference in favor of one alternative based on this value.*
+
+The Cartesian product of criteria scales is denoted by  X = $$\prod_{j=1}^{n}$$ $$X_j$$ in which $$X_j$$ represents all the possible values of a criterion j. 
+We designate $$g_j$$ : $$A$$ $$\to$$ $$X_j$$ the function assigning a value to each alternative of the set $$A$$. $$A$$ represents a finite set of alternatives for which a ranking or a sorting has to be determined. Let $$a$$ $$\in$$ $$A$$ be an alternative $$a$$ contained in the set $$A$$. We denote by $$a_j$$ = $$g_j(a)$$ the performance of an alternative a on criterion $$j$$. The performance vector associated to each alternative a is denoted by $$a$$ = ($$a_1$$, ..., $$a_n$$);
+
+There exist different **types of preference relation** between two alternatives. Roy and Bouyssou (1993) list four types of preferences:
```

Comment:
Should we talk about that here? As we don't need it for our algorithm

Reply:
Doesn't it give general information on Mcda models ?